Tests/js/views/services/assets/ez-locationviewviewservice-tests.js

```diff
@@ -430,6 +430,216 @@ YUI.add('ez-locationviewviewservice-tests', function (Y) {
             Y.Mock.verify(app);
             Y.Mock.verify(contentMock);
         },
+
+        "Should launch the universal discovery widget when receiving an moveAction event": function () {
+            this.service.on('contentDiscover', function (e) {
+                Y.Assert.isObject(e.config, "contentDiscover config should be an object");
+                Y.Assert.isFunction(e.config.contentDiscoveredHandler, "config should have a function named contentDiscoveredHandler");
+            });
+            this.service.fire('whatever:moveAction');
+        },
+    });
+
+    moveContentTest = new Y.Test.Case({
+        name: "eZ Location View View Service move content tests",
+
+        setUp: function () {
+            var that = this;
+            this.capiMock = new Y.Test.Mock();
+            this.locationMock = new Y.Test.Mock();
+            this.responseMock = new Y.Test.Mock();
+            this.contentServiceMock = new Y.Test.Mock();
+            this.locationId = 'location/2/2/2/2';
+            this.parentLocationId = 'location/1/2/3/4';
+            this.finalLocation = 'finalLocation/4/5/6/7';
+            this.error = false;
+
+            Y.Mock.expect(this.responseMock, {
+                method: 'getHeader',
+                args: ['location'],
+                returns: this.finalLocation
+            });
+
+            Y.Mock.expect(this.capiMock, {
+                method: 'getContentService',
+                returns: this.contentServiceMock
+            });
+
+            Y.Mock.expect(this.locationMock, {
+                method: 'get',
+                args: ['id'],
+                returns: this.locationId
+            });
+
+            Y.Mock.expect(this.locationMock, {
+                method: 'move',
+                args: [Y.Mock.Value.Object, this.parentLocationId, Y.Mock.Value.Function],
+                run: function (options, parentLocationId, callback) {
+                    options.api = that.capiMock;
```

Comment:
??? Maybe you need to add an assertion here but for sure, you should not assign anything.